Use partCounter in question parts

bschne opened the pull request:
This change uses the "partCounter" when working with either \alphaparts or \arabicparts question parts. This allows for splitting the list of question parts with some additional question text in between (i.e. using more than one \alphaparts or \arabicparts environment within a question) while still getting a continued enumeration label within the same question.

![image](https://user-images.githubusercontent.com/18404509/56867839-b1565680-69ea-11e9-88d5-58aa1a2e098f.png)

Before this change, the second list would have started over with (a), (b), ...

jez commented:
Thanks for this change!

I'm still a little unclear on how to use this. Would you mind updating the sample file `homework.tex` with an example? Probably in a new section, but potentially in an existing section.

Also if you could recompile and commit the `homework.pdf` that'd be great too.

bschne commented:
Hi @jez 

I just updated the homework.tex file with a new question demoing the functionality. In addition, I switched the other `alphaparts` and `arabicparts` environments from using `\item` for the parts to `\questionpart` as this is now needed to increment the counter (missed that in my prev. commit).

Recompiled PDF is included.

jez commented:
Looks great, thanks!
